deploy.sh

```diff
@@ -13,8 +13,8 @@ if [ -f .env ]; then
 fi
 
 # Configuration with defaults
-BUCKET_NAME="${S3_BUCKET:-loopkit-js-cdn}"
-SDK_PATH="${SDK_PATH:-javascript}"  # Organize by SDK type (configurable)
+BUCKET_NAME="${S3_BUCKET:-loopkit-public-cdn}"
```

Comment:
Reason why you changed these?

Reply:
The `loopkit-js-cdn` bucket doesn't exist and the current SDK path within the` loopkit-public-cdn` bucket is `js`.

Reply:
you are correct i just dont know how this was working for me lol